packages/visual-dev/src/components/ObjectFieldTemplate/ObjectFieldTemplate.stories.tsx

```diff
@@ -66,6 +66,65 @@ export const Default = () => {
   );
 };
 
+export const TwoColumns = () => {
```

Comment:
Would be nice if we could add a bit of a gap here to make the visual look a bit better in storybook.